src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -482,15 +482,24 @@ void ResumeCtrlImpl::StartAppHmiStateResumption(
     LOG4CXX_DEBUG(logger_, "No applicable HMI level found for resuming");
     return;
   }
-
   const bool is_resume_allowed_by_low_voltage =
       CheckLowVoltageRestrictions(saved_app);
 
   const bool is_hmi_level_allowed_by_ign_cycle =
       CheckIgnCycleRestrictions(saved_app);
 
-  const bool restore_hmi_level_allowed =
-      is_resume_allowed_by_low_voltage && is_hmi_level_allowed_by_ign_cycle;
+  const bool is_app_revoked =
+      application_manager_.GetPolicyHandler().IsApplicationRevoked(
+          application->policy_app_id());
+
+  const mobile_apis::HMILevel::eType app_hmi_level = application->hmi_level();
+  const bool hmi_level_allow =
```

Comment:
@IGapchuk you can remove this check for allowed for resumption HMI level as it is already done in `CheckAppRestrictions()` before that. Just keep `is_app_revoked`

Reply:
@AKalinich-Luxoft  Fixed in ec314ff446e9674f40a55a162cdc239a328613ec